usaspending_api/api_docs/api_documentation/award_spending/award_category.md

```diff
@@ -0,0 +1,65 @@
+## [Award Spending Award Category](#usaspending-api-documentation)
```

Comment:
We are not using this endpoint.

You should remove this endpoint from the documentation **and the API**.

Reply:
done